sadcowpng/sharedspaces/views.py

```diff
@@ -320,7 +318,7 @@ def update_space(request, space_id):
     return render(request, 'sharedspaces/update_space.html', context=context)
 
 
-#@user_is_space_owner
+# @user_is_space_owner
```

Comment:
This is supposed to be uncommented and the decorator top of update date and time should be commented but that will be fixed with my feature so no worries.